### PR TITLE
[replication] Added PowerScale replication section

### PR DESCRIPTION
**Pull request by walker2**
# Description

- Added PowerScale replication section
- Fixed a couple of formatting issues
- Added PowerScale actions descriptions

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/116 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by maxan98 — APPROVED** [review]

**Comment by walker2** [comment]
> Link is broken in storage classes

Thanks, fixed the broken link

**Review by maxan98 — APPROVED** [review]

File: content/docs/replication/deployment/powerscale.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+To be able to provision replicated volumes you need to create properly configured storage classes on both source and target clusters.
+
+A pair of storage classes on the source and target clusters would be essentially `mirrored` copies of one another.
+You can create them manually or with help from `repctl`.
+
+#### Manual Storage Class Creation
+
+You can find sample replication enabled storage class in the driver repository
+at `./samples/storageclass/isilon-replication.yaml`.
```
Comment (shanmydell) [inline content/docs/replication/deployment/powerscale.md:94]: @walker2 : Please paste the full yaml link

File: content/docs/replication/deployment/powerscale.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+After figuring out how storage classes would look, you just need to go and apply them to your Kubernetes clusters with `kubectl`.
+
+#### Storage Class Creation With repctl
+
+`repctl` can simplify storage class creation by creating a pair of mirrored storage classes in both clusters
+(using a single storage class configuration) in one command.
+
+To create storage classes with `repctl` you need to fill up the config with necessary information.
+You can find an example in `repctl/examples/powerscale_example_values.yaml`, copy it, and modify it to your needs.
```
Comment (shanmydell) [inline content/docs/replication/deployment/powerscale.md:142]: Please give a link

**Review by coulof — APPROVED** [review]
I committed a few changes directly to this branch.
Hope that's fine for you.

File: content/docs/replication/deployment/powerscale.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+---
+title: PowerScale
+linktitle: PowerScale
+weight: 7
+description: >
+  Enabling Replication feature for CSI PowerScale
+---
+## Enabling Replication In CSI PowerScale
+
+For the Container Storage Modules (CSM) for Replication sidecar container to work properly it needs to be installed
```
[the remark this answers is not in the document]
Comment (walker2) [reply content/docs/replication/deployment/powerscale.md]: Sure, rephrased those sections for both PScale and PStore

**Review by coulof — APPROVED** [review]
lgtm

**Review by maxan98 — APPROVED** [review]

**Review by AronAtDell — DISMISSED** [review]

**Review by shanmydell — APPROVED** [review]

**Review by dmpost — APPROVED** [review]

**Review by maxan98 — APPROVED** [review]

**Pull request merged**